assets/in

```diff
@@ -74,9 +74,6 @@ if [ "$skip_download" = "false" ]; then
     --entrypoint /tmp/print-metadata  \
     "$image_name" > ${destination}/metadata.json
 
-  mkdir -p ${destination}/rootfs/
-  docker export $(cat /tmp/container.cid) | tar -xf - -C ${destination}/rootfs/
```

Comment:
Admittedly it's not documented at all - there's a filesystem schema assumed by `image_resource`. https://github.com/concourse/atc/blob/102e8856983a548797f5c55ebe4f2deea7540ef2/worker/worker.go#L220-L232